File: Software Technologies/JavaScript/LAB/Largest 3 Numbers.js
function getLargestThree(arr) {
    let numbers = arr.map(Number);

    numbers.sort((a,b)=>b-a);
    let loopLength = numbers.length<3?numbers.length:3;
    for(let i=0;i<loopLength;i++)
        console.log(numbers[i]);
}

File: Software Technologies/Java/LAB/Largest 3 Numbers.java
package com.company;

import java.util.*;

public class Main {
    static Scanner Input = new Scanner(System.in);

    public static void main(String[] args) {
        String[] input = Input.nextLine().split("\\s+");
        ArrayList<Integer> numbers = new ArrayList<Integer>();
        for(int i=0;i<input.length;i++)
            numbers.add(Integer.parseInt(input[i]));
        numbers.sort(Integer::compareTo);
        Collections.reverse(numbers);

        int loopLen = numbers.size()>=3?3:numbers.size();
        for(int i=0;i<loopLen;i++)
            System.out.println(numbers.get(i));
    }
}